cli/.js/src/main/scala/aqua/RunCommand.scala

```diff
@@ -38,34 +44,80 @@ object RunCommand extends Logging {
    * @param air code to call
    * @return
    */
-  def funcCall(multiaddr: String, air: Generated, config: TransformConfig)(implicit
+  def funcCall(multiaddr: String, air: String, functionDef: FunctionDef, config: RunConfig)(implicit
     ec: ExecutionContext
-  ): Future[Validated[String, Unit]] = {
+  ): Future[Unit] = {
     (for {
       _ <- Fluence
-        .start(multiaddr)
+        .start(PeerConfig(connectTo = multiaddr))
         .toFuture
       peer = Fluence.getPeer()
+      promise = Promise.apply[Unit]()
       _ = CallJsFunction.registerUnitService(
         peer,
-        "console",
-        "print",
-        args => println("print: " + args)
+        config.consoleServiceId,
+        config.printFunctionName,
+        args => {
+          // if an input function returns a result, our success will be after it is printed
+          // otherwise finish after JS SDK will finish sending a request
+          println("result: " + args)
```

Comment:
I think there's no need for `result:` prefix, really. But I'm not sure. Do you have a screenshot of execution?